src/Compilers/CSharp/Portable/Symbols/Synthesized/InterpolatedStringBuilderLocalSymbol.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// A synthesized local variable.
+    /// </summary>
+    [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
+    internal sealed class InterpolatedStringBuilderLocalSymbol : LocalSymbol
```

Comment:
>InterpolatedStringBuilderLocalSymbol [](start = 26, length = 36)

Is this mostly a clone of ```SynthesizedLocal``` type? In that case I would prefer us to either inherit from ```SynthesizedLocal```, or inherit from a common base extracted from ```SynthesizedLocal``` without changing style for properties to simplify the review process.